liquibase-couchbase/src/test/java/liquibase/ext/couchbase/change/MutateInChangeTest.java

```diff
@@ -97,7 +116,22 @@ void Should_generate_statement_correctly_with_where() {
     }
 
     private LiquibaseMutateInSpec spec(String path, String value, DataType dataType, MutateInType type) {
-        return new LiquibaseMutateInSpec(path, Arrays.asList(new Value(value, dataType)), type);
+        return new LiquibaseMutateInSpec(path, Collections.singletonList(new Value(value, dataType)), type);
+    }
+
+    private MutateInChange change(List<LiquibaseMutateInSpec> specs) {
```

Comment:
Maybe `createTestChange()` will be more obvious here?

Reply:
changeWithQuery should be more obvious